waiter/test-files/saml/idp-redirect-location.txt

```diff
@@ -0,0 +1 @@
+https://idp-host/idp-endpoint?SAMLRequest=fZHNTsMwEITvPIW1dzdpC4haTapAQUQqULUJSNxMYlFLzTp4N4XHJz9UKpfebGtndubzfPFT7cXBeLIOIxiPQhAGC1da%2FIwgzx7kDSziiznpal%2BrpOEdbsxXY4hFQmQ8t7I7h9RUxm%2BNP9jC5JtVBDvmmlQQ7Bwx6soE39qy8VK3DkFnFuiCQCxbI4ua%2B%2BVHjS1r2en6g8GydhYZRLqM4C1Js%2FuNzPN02T4QNSZFYo0cwSQcz2R4JcNpNr5Wl1M1nb2DWHvHrnD7W4tDpcajcposqS4WKS7UNnlaqckoVB%2FDEKnHLFvL9cs26w0OtjT%2BuZ2OYGgB4vUIbNIBaxEiqR7R%2BQX1XxqIe6Cqb%2BBP9Ofl%2Bkgc4iHIPDixiYfb%2F1%2BKfwE%3D&RelayState=ezpob3N0ICJteS5hcHAuZG9tYWluIiwgOnJlcXVlc3QtdXJsICJodHRwOi8vbXkuYXBwLmRvbWFpbi9teS1lbmRwb2ludD9hPTEmYj1jIiwgOnNjaGVtZSAiaHR0cCJ9
```

Comment:
Why do we need this saved in a file instead of generating this in the test?

Reply:
done